sdk/keyvault/keyvault-keys/review/keyvault-keys.review.api.md

```diff
@@ -0,0 +1,295 @@
+## API Report File for "@azure/keyvault-keys"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { HttpClient } from '@azure/core-http';
+import { HttpPipelineLogger } from '@azure/core-http';
+import * as msRest from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PageSettings } from '@azure/core-paging';
+import { ServiceClientCredentials } from '@azure/core-http';
+import { ServiceClientOptions } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+// @public
+export interface CreateEcKeyOptions extends CreateKeyOptions {
+    curve?: JsonWebKeyCurveName;
+    hsm?: boolean;
+}
+
+// @public
+export interface CreateKeyOptions {
+    enabled?: boolean;
+    expires?: Date;
+    keyOps?: JsonWebKeyOperation[];
+    // (undocumented)
+    keySize?: number;
+    notBefore?: Date;
+    requestOptions?: msRest.RequestOptionsBase;
+    tags?: {
+        [propertyName: string]: string;
+    };
+}
+
+// @public
+export interface CreateRsaKeyOptions extends CreateKeyOptions {
+    hsm?: boolean;
+    keySize?: number;
+}
+
+// @public
+export class CryptographyClient {
+    constructor(url: string, key: string | JsonWebKey, // keyUrl or JsonWebKey
+    credential: TokenCredential, pipelineOrOptions?: ServiceClientOptions | NewPipelineOptions);
+    protected readonly credential: ServiceClientCredentials | TokenCredential;
+    decrypt(algorithm: JsonWebKeyEncryptionAlgorithm, ciphertext: Uint8Array, options?: DecryptOptions): Promise<DecryptResult>;
```

Comment:
Why do these take only a Uint8Array? I would expect this could take a NodeJS buffer, an ArrayBuffer, and a blob (in browsers)?

Reply:
I would defer to your expertise in this area, as to what is idiomatic to js. However, if it makes a difference in this case the value, at least until we support multiblock encryption will be fairly small as in  < 1K

Reply:
ArrayBuffer | buffer would be a better choice here. This should also allow Uint8Array.  We should investigate and also check what crypto.js does.